api/views/user_specific_module_maintenance.py

```diff
@@ -592,3 +573,23 @@ def get_repo(repo_url: str, owner: str, repo_name: str) -> RepoUtil:
                 description='bad request - could not clone the Github repository. Please check owner,'
                 ' repository and path of the request - {}'.format(e.stderr),
             )
+
+
+def get_response(correlation_id: str) -> str:
```

Comment:
I believe this function would better be placed in the `Job_runner.py` because this functionality isn't limited to only `user_specific_module_maintenance.py`

Reply:
Are there any other places where it could be useful?

Reply:
I mean in the future it's possible to see the status of a job, so Maybe in the future, we will need it somewhere else, and it's not obvious that this functionality is placed in the `user_specific_module_maintenance.py` module, I would expect it to be placed in the same module as the runner itself